File: app/src/main/java/com/prance/teacher/utils/ModelUtil.java
package com.prance.teacher.utils;

/**
 * Created by bingbing on 2016/10/15.
 */

public class ModelUtil {

    /**
     * 是否是测试模式
     *
     * @return
     */
    public static boolean isTestModel() {
        try {
            String msg = MetaUtil.getMetaValue(Constants.TEST_SETTING);
            if (msg.equals(Constants.OPEN)) {
                return true;
            } else {
                return false;
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
        return false;
    }
}

File: lib.common/src/main/java/com/prance/lib/common/utils/ModelUtil.kt
package com.prance.lib.common.utils

/**
 * Created by bingbing on 2016/10/15.
 */

object ModelUtil {

    /**
     * 是否是测试模式
     *
     * @return
     */
    val isTestModel: Boolean
        get() {
            try {
                val msg = MetaDataUtil.getMetaDataInApp(Constants.TEST_SETTING)
                return msg == Constants.OPEN
            } catch (e: Exception) {
                e.printStackTrace()
            }

            return false
        }
}
